content/doc/developer/publishing/continuous-integration.adoc

```diff
@@ -8,12 +8,24 @@ It will build all plugin repositories in the `jenkinsci` organization that have
 
 The typical plugin build (Maven or Gradle) can be run by just having the following statement in the `Jenkinsfile`:
 ----
-buildPlugin()
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17], // use 'docker' if you have containerized tests
+    [platform: 'windows', jdk: 11],
+])
 ----
 
 Gradle support in `buildPlugin()` is deprecated and will be eventually removed. Please use:
 ----
-buildPluginWithGradle()
+buildPluginWithGradle(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17], // use 'docker' if you have containerized tests
+    [platform: 'windows', jdk: 11],
+])
 ----
 
+NOTE: If the Jenkinsfile configuration is not set properly, you will receive a low-level class version error.
```

Comment:
This is correct but a bit ambiguous as to what specific conditions will cause the error. Perhaps we want to make this more specific?

```suggestion
NOTE: If the `Jenkinsfile` configuration includes Java 8, you will receive a low-level class version error when using version 4.52 or later of the plugin parent POM.
```

If we think that is too specific and want to keep the tone of the documentation at a higher-level, I am fine with dropping this suggestion. But since this is documentation for developers I think it is probably fine to go into more detail.

Reply:
That is a great suggestion Basil, and I agree that it helps provide specifics on this, which is important.